.azure-pipelines/integration-tests.yml

```diff
@@ -329,7 +329,7 @@ jobs:
       projects: |
         test/test-applications/integrations/**/*.csproj
         !test/test-applications/integrations/dependency-libs/**/*.csproj
-      arguments: --configuration $(buildConfiguration) -p:Platform=$(buildPlatform) -p:ManagedProfilerOutputDirectory=$(publishOutput) -p:BuildInParallel=false /nowarn:netsdk1138
+      arguments: --configuration $(buildConfiguration) -p:Platform=$(buildPlatform) -p:ManagedProfilerOutputDirectory=$(publishOutput) /nowarn:netsdk1138 -p:ExcludeManagedProfiler=true -p:ExcludeNativeProfiler=true -p:LoadManagedProfilerFromProfilerDirectory=false
```

Comment:
Oops, yep. I guessed that this wouldn't be necessary now that we're not copying things around and figured it would speed up the build, but no point risking it I guess!

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1171/commits/e0b070b25e6ea9301cf9dbed9f9b15b6a52855ed